api/utils/hysds_util.py

```diff
@@ -624,7 +624,7 @@ def get_mozart_queues():
         raise Exception("Couldn't get list of available queues")
 
 
-def get_mozart_jobs(username, page_size=10, offset=0):
+def get_mozart_jobs(username, **kwargs):
```

Comment:
This should not be kwargs. Please define the parameters accepted here explicitly so that we can keep track of the changing mozart api with different hysds versions.

Reply:
Updated function signature with explicit list of params.